setup.py

```diff
@@ -87,7 +97,6 @@
 # present, then it will still attempt to use Cython.
```

Comment:
Should we remove Python 2 classifiers @ line 84-85? And add new ones.

Reply:
Good catch. I removed these earlier. This must have been a rebase mistake.